calculator.go

```diff
@@ -1,13 +1,125 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes a list of numbers and returns the result of adding them together.
+func Add(nums ...float64) float64 {
+	switch len(nums) {
+	case 0:
+		return 0
+	case 1:
+		return nums[0]
+	default:
```

Comment:
So if we remove the special cases, we don't even need the `switch` statement, do we?